src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/RequestLogs.java

```diff
@@ -11,11 +11,11 @@ public class RequestLogs {
     @Field("clientRequestId")
     public String clientRequestId;
 
-    @Field("requestId")
-    public String requestId;
+    @Field("gatewayRequestId1")
+    public String gatewayRequestId1;
 
-    @Field("gatewayRequestId")
-    public String gatewayRequestId;
+    @Field("gatewayRequestId2")
+    public String gatewayRequestId2;
 
     @Field("response")
     public String response;
```

Comment:
Why do we need `response` and `rawResponse` as two fields?

Reply:
the response is the status code of the request made to the ABDM gateway after linking care context, such that the client/facility can check the status. when it comes to rawResponse we are dumping the data into wrappersDB

Reply:
i am open to change the code, we can also dump the on-Add-careContext response into wrapperDb and from there we can fetch the status of linking.

Reply:
Modified the naming convention